lib/topological_inventory/ansible_tower/operations/processor.rb

```diff
@@ -1,25 +1,133 @@
 require "topological_inventory/ansible_tower/logging"
+require "topological_inventory-api-client"
+require "topological_inventory/ansible_tower/operations/core/ansible_tower_client"
+require "topological_inventory/ansible_tower/operations/core/topology_api_client"
 
 module TopologicalInventory
   module AnsibleTower
     module Operations
       class Processor
         include Logging
+        include Core::TopologyApiClient
+
+        SLEEP_POLL = 10
+        POLL_TIMEOUT = 1800
 
         def self.process!(message)
-          new(message).process
+          model, method = message.headers['message_type'].to_s.split(".")
+          new(model, method, message.payload).process
         end
 
-        def initialize(message)
-          self.message = message
+        # @param payload [Hash] https://github.com/ManageIQ/topological_inventory-api/blob/master/app/controllers/api/v0/service_plans_controller.rb#L32-L41
+        def initialize(model, method, payload)
+          self.model           = model
+          self.method          = method
+          self.params          = payload["params"]
+          self.identity        = payload["request_context"]
         end
 
         def process
-          # TODO: handle the operation
+          logger.info("Processing #{model}##{method} [#{params}]...")
+          result = order_service(params)
```

Comment:
We're going to need to handle more operations in the future so dynamically building the method instead of just calling order for any message received would be much better